### ISSUE TITLE
--playlist-sync option runs?

### ISSUE THREAD

**Issue by Pepetops** [jrnewell/spotify-ripper#182]
Hi again. I download a dynamic list with songs that up and down on list...
I was thinking that with playlist-sync spotify-ripper only rename the file {idx}-blabla... with the new {idx} but I see I have two files, f.e. 006-old-file.mp3 and 006-new-file.mp3... And also lost time downloading the song I have but with another {idx}...
It's normal?
Regards

**Comment by jrnewell** [comment]
No, it should just rename file and not redownload the song (assuming you don't have the --overwrite option set).  I had trouble replicating the issue but I found a regression bug that might be the cause.  Can you try again with version ``2.8.3``?

**Comment by Pepetops** [comment]
Hi. I use now 2.8.3 version. 
To download list I use this options: 
-c -b 192 --flat-with-index --playlist-m3u --playlist-sync -d Spotify_Global
Well. I see the program deletes all files that I suppose not in list now and then begins download the files... Not rename...

**Comment by xtwistedmetal** [comment]
My playlist I made to try and rip, it just keeps trying to load the playlist and it returns the download size. Then it says attempt 1 failed Spotify returned None for playlist, trying again. and it does it 3 times and fails, any suggestions?

**Comment by Pepetops** [comment]
No... Could you send a copy of messages you see?

**Comment by jrnewell** [comment]
I will try to investigate this more when I have some time...

**Comment by xtwistedmetal** [comment]
```
Logged in as XXX
Loading playlist...
Total Download Size: 1.63 GB
Attempt 1 failed: Spotify returned None for playlist, trying again in 5 seconds...
Attempt 2 failed: Spotify returned None for playlist, trying again in 5 seconds...
Attempt 3 failed: Spotify returned None for playlist, trying again in 5 seconds...
Could not load playlist...
Logging out...
```
That is what it does while running 
spotify-ripper -u XXX -p XXX-k /home/musix/Desktop/spotify_appkey.key -d /home/USER/Desktop/musixSpot spotify:user:username:playlist:00CjiLWyKpfSwmzgS5sl6z , I was trying to just copy the play list into a txt and have it load a txt file with all the spotify track urls but it says no results =/
